composer.json

```
@@ -1,14 +1,20 @@
 {
+	"config": {
+		"platform": {
+			"php": "8.0"
+		}
+	},
 	"require": {
 		"vgrem/php-spo": "2.4.*"
 	},
 	"scripts": {
 		"cs:check": "php-cs-fixer fix --dry-run --diff",
 		"cs:fix": "php-cs-fixer fix",
-		"lint": "find . -name \\*.php -not -path './vendor/*' -not -path './build/*' -print0 | xargs -0 -n1 php -l"
+		"lint": "find . -name \\*.php -not -path './vendor/*' -not -path './build/*' -print0 | xargs -0 -n1 php -l",
+		"test:unit": "phpunit -c phpunit.xml --fail-on-warning  --fail-on-risky --color"
 	},
 	"require-dev": {
 		"phpunit/phpunit": "^9.5",
-		"nextcloud/coding-standard": "^0.5.0"
+		"nextcloud/coding-standard": "1.0"
```

```suggestion
		"nextcloud/coding-standard": "^1.0"
```

to get future updates in case we bump to 1.1 or the alikes?

---

yeah, copied it from elsewhere, was wondering the same.